mops/benchmarks/opsa.cpp

```diff
@@ -0,0 +1,30 @@
+#include <algorithm>
+#include <vector>
+
+#include "mops.hpp"
+#include "utils.hpp"
+
+int main() {
+
+    auto A = std::vector<double>(60000 * 13);
+    fill_vector_random_floats(A);
+
+    auto B = std::vector<double>(60000 * 20);
+    fill_vector_random_floats(B);
+
+    auto indices_output = std::vector<int>(60000);
```

Comment:
```suggestion
    auto indices_output = std::vector<int32_t>(60000);
```

Reply:
Are you sure? This is in the benchmarks, not in the core library